docs/content/latest/admin/yb-admin.md

```diff
@@ -1161,14 +1161,14 @@ To verify if any tables are already configured for replication, use [`list_cdc_s
 yb-admin \
     -master_addresses <master-addresses> \
     setup_universe_replication \
-    <producer_universe_uuid> \
-    <producer_master_addresses> \
+    <source_universe_uuid> \
+    <cource_master_addresses> \
     <comma_separated_list_of_table_ids>
 ```
 
 * _master-addresses_: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
-* *producer_universe_uuid*: The UUID of the producer universe.
-* *producer_master_addresses*: Comma-separated list of master producer addresses.
+* *soruce_universe_uuid*: The UUID of the producer universe.
```

Comment:
'soruce' typo

'producer universe' - should that now be changed to source? likewise 'master producer addresses' below

Reply:
Changed